.github/workflows/binary-check.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: coursier/cache-action@v6.3
 
       - name: Install Adoptium Temurin OpenJDK
-        uses: coursier/setup-action@v1.2.0
+        uses: coursier/setup-action@v1.2
```

Comment:
Looks like we can use `v1` now. 
https://github.com/coursier/setup-action/issues/163

Reply:
:+1: Updates, thanks!